### PR TITLE
Fix tif naming

### LINKED ISSUE #65: MNT: adjust current filename naming scheme for ``data_field``
current file name would contain: ``<motor_name>=<readback_value>`` for example, ``temperature=300``.

However, based user feedback, this would introduce difficulties when using file-based analysis software. We might simply record the readback value.

### PR DESCRIPTION
**Pull request by chiahaoliu**
closes #65 

@sbillinge @CJ-Wright  ready for review

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Should we publish a cheat sheet with all the files in a header which states what the keys for the filename values are?

**Comment by chiahaoliu** [comment]
Do you mean a reference table for possible data keys?

**Comment by CJ-Wright** [comment]
I meant that we could write a file `keys.txt` which has the keys that were used to build the filename in order. That way if the user wants to find what metadata goes to which file they can look at the cheat sheet to see what they were. 

I am suggesting this because in the past, if I am reading the code correctly, we used to give `key=value` in the filename, right? Now we are just giving `value` so we might want to give users a way to figure out what key goes to the values. I also assume that we use the same keys for a given header/save session so we only need to make one file per save/header.

File: xpdan/data_reduction.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def _feature_gen(self, event):
         for key in self.data_fields:
             val = event['data'].get(key, None)
             if el is not None:
-                feature = "{}={}".format(key, val)
+                feature = "{}".format(val)
```
Comment (sbillinge) [inline xpdan/data_reduction.py]: can you quickly write a test for this so that it passes all the integration coveralls tests?

Comment (CJ-Wright) [inline xpdan/data_reduction.py]: We need to add a temperature or some other variable into the `insert_imgs` function.

Comment (chiahaoliu) [inline xpdan/data_reduction.py]: thanks for merging it. I was working on the test and not aware of it. Should we still finish this test?

Comment (CJ-Wright) [inline xpdan/data_reduction.py]: I mean it would be a good test/functionality to have, do you mind putting it into a separate PR?

Comment (chiahaoliu) [inline xpdan/data_reduction.py]: yes, another thorough PR sounds a better idea.
